unity-client/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/AssetCatalog.cs

```diff
@@ -0,0 +1,128 @@
+using DCL;
+using DCL.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AssetCatalog : MonoBehaviour
```

Comment:
This is acting as bridge right? I think we should rename this to `AssetCatalogBridge` to reflect its behaviour.

Also, we should make a test suite covering the use cases of this one.

Reply:
I'm sure this can be confusing taking into account we have a lot of bridges having the `Controller` name. We are currently in process of migrating all the `NameController` classes with bridge responsibility to `NameBridge`.

Reply:
The name has changed, regarding the test, I will add it to the list of test that I need to create, but we need to group up to see  what is the direction of the test